src/components/forms/charcount.scss

```diff
@@ -0,0 +1,13 @@
+@import "../../colors";
+
+p {
```

Comment:
Needs the `.char-count` class, otherwise it applies to all `p`'s on the page if a `CharCount` component is also on the page.